CONTRIBUTING.md

```diff
@@ -0,0 +1,13 @@
+# Contributing to the Rails Girls Summer of Code website
+
+We're always very happy about contributions to our website, so thank you for taking the time to contribute! Here you will find some guidelines about what is expected from contributors. Before getting started, please make sure you read and abide by our [Code of Conduct](https://github.com/rails-girls-summer-of-code/summer-of-code/blob/gh-pages/CODE_OF_CONDUCT.md).
```

Comment:
I wouldn't mind if we don't ask to 'abide'. In the sentence before, it is clearly stated that that is what is expected from contributors.